docs/products/cassandra/howto/enable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,199 @@
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select an Aiven for Apache Cassandra service on which you'd like to enable CCR.
+3. In the **Overview** tab of your service's page, navigate to **Cross Cluster Replication** and select **Migrate to Cross Cluster**.
+
+   .. note::
+      
+      When you enable CCR on a particular service, you create another service with the same plan, number of nodes, and disk storage.
+
+4. In the **Create a Cross Cluster Replica** view
+   
+   1. Select a cloud provider and a cloud region, define a name for your new service, and select **Continue**.
+   2. Examine the **Service Summary** section and make sure the configuration for your new service meets your expectations. If so, select **Create Cross Cluster Replica**. 
+
+.. topic:: Result
+   
+   CCR has been enabled by connecting your service to another new service, which is now visible in the **Overview** tab under **Cross Cluster replication**.
+
+Enable CCR with CLI
+-------------------
+
+You can enable CCR for your new or existing Aiven for Apache Cassandra service using CLI.
+
+.. note::
+   
+   In this instruction, the :doc:`Aiven CLI client </docs/tools/cli>` is used to interact with Aiven APIs.
+
+.. topic:: Understand parameters to be supplied
+
+   * ``service_to_join_with`` parameter value needs to be set to a name of an existing service in the same project. The supplied service name indicates the service you connect to for enabling CCR. The two connected services create a CCR service pair.
+   * ``cassandra.datacenter`` parameter value needs to be set to a name of a datacenter for your service. Make sure each of the two service constituting a CCR pair belongs to a different datacenter.
+
+On a new service
+''''''''''''''''
+
+Use the :ref:`avn service create <avn-cli-service-create>` command to create a new service. Add the ``service_to_join_with`` and ``cassandra.datacenter`` parameters and set their values as needed.
+
+.. code-block:: bash
+
+   avn service create                                   \
+      --service-type cassandra                          \
+      --cloud cloud_region_name                         \
+      --plan service_plan_name                          \
+      -c cassandra.datacenter=datacenter_name           \
+      -c service_to_join_with=existing_service_name     \
+      service_name
+
+On an existing service
+''''''''''''''''''''''
+
+.. important::
+
+    * Your existing service has ``NetworkTopologyStrategy`` set as a replication strategy for its keyspace.
+    * Your existing service and the service you connect to (``service_to_join_with``) are hosted on different datacenters.
+
+Use the :ref:`avn service update <avn-cli-service-update>` command to modify your service configuration by adding the ``service_to_join_with`` parameter and set its value as needed.
+
+.. code-block:: bash
+
+   avn service update service_name                     \
```

Comment:
I think there's a misunderstanding about what it means to enable CCR on an existing service. It means "I have a service X, I'm creating a new empty service Y and setting it up to join with X (replicate from X)". That's what happens if you do this in the UI. 

On the other hand, the command you've added here means "I have a service X, with some data, and a service Y, with probably some other data; now I command Y to join X (start replicating from X)". This setup is not supported and is supposed to give you an error; if this command has not errored out, it's a bug.

Reply:
maybe it would be better if we go "Create a new CCR service pair/Add a CCR peer in another region to an existing service" instead of "Enable CCR for a new service/existing service", this would stress the fact that there's two services involved

Reply:
Thanks, Dmitry :-) Does it mean that for the CLI method:
1. For the "Create a new CCR service pair" scenario: We need run two commands - (1) create a regular new service (2) create another new service with `service_to_join_with=service_1_name` specified?
2. For the "Add a CCR peer in another region to an existing service" scenario: We need only one command - create a new service with `service_to_join_with=existing_service_name` specified?

Reply:
Also, am I right I need to same approach for the API method instruction? @dmitry-potepalov